src/mlj_model_interface.jl

```diff
@@ -53,74 +55,61 @@ end
 
 
 # # FIT AND  UPDATE
+include("fit_utils.jl")
+include("entity_embedding_utils.jl")
```

Comment:
I prefer to avoid includes that don't appear in at the top level (ie in Flux.jl).

Reply:
OK. Will take them to top-level.